readthedocs/integrations/utils.py

```diff
@@ -12,12 +17,12 @@ def normalize_request_payload(request):
     :returns: The request body as a string
```

Comment:
Please, update the docstring to match the new signature and also the new behavior.